_security/access-control/read-only-role.md

```diff
@@ -0,0 +1,41 @@
+---
+layout: default
+title: Read only roles
+parent: Access control
+nav_order: 150
+---
+
+# Read only roles
+
+As with any roles in OpenSearch, read only roles can be configured using three methods: `yml` configuration files, API and OpenSearch Dashboards. The most user friendly approach for anyone getting familiar with Roles and role mappings is to use OpenSearch Dashboards, as it is easy to navigate the creation of roles and assignment to users. The basic steps of creating roles/mappings/users are outlines here (https://opensearch.org/docs/latest/security/access-control/users-roles/)
+
+## Basic read only role
+
+If you are looking to create a role to access OpenSearch Dashboards, view existing dashboards/visualizations and query different indexes, assuming you want the user to have read access to all indexes and tenants, you can add cluster permission group of `cluster_composite_ops_ro`, `read` access to all indexes (`*`) and `read` access to all tenants
+
+![creating role]({{site.url}}{{site.baseurl}}/images/role_creation_read_only.png)
+
+Once the role is created, you can directly map the role to a user by going to "Mapped users" tab in roles, select "Map users" and select the user to map to this role. 
+
+![mapping users]({{site.url}}{{site.baseurl}}/images/mapping-users.png)
+
+## OpenSearch Dashboards readonly_mode
+
+OpenSearch Dashboards comes with functionality to only give access to `Dashboards` UI by using readonly_mode.
```

Comment:
Not sure on stylization of "readonly_mode;" does it need to be in quotes?

Reply:
I've added code to readonly_mode